packages/credit-protocol/index.ts

```diff
@@ -418,15 +418,31 @@ export default class CreditProtocol {
     return config.gasPrice
   }
 
-  async getTxCost(currency: string, gasNeeded: number) {
+  async getTransactionCosts(currency: string, gasNeeded: number) {
     try {
       const gasPrice = await this.getGasPrice()
       const rate = await this.getEthExchange(currency)
+      const weiCost = gasPrice * gasNeeded
+      const ethCost = weiCost / WEI_PER_ETH
+      const currencyCost = Math.max( 0.01, ethCost * Number(rate) )
+
+      return {
+        ethCost,
+        ethCostFormatted: `${ethCost}`.slice(0,6),
```

Comment:
Magic number `6` here. Might there be a better way to achieve whatever this line is doing?

Reply:
The goal here is to clarify the previous code, which looked like this:
```
return `${Math.max( 0.01, gasPrice * Number(rate) * gasNeeded / WEI_PER_ETH )}`.slice(0,6)
```

Reply:
I don't like it either... we should introduce constants for all of our significant digits. Future update...